### PR TITLE
Autoupdating Homebrew Formula on Release

### PR DESCRIPTION
**Pull request by alpreu**
Resolves https://github.com/xcodeswift/xctools/issues/11

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -39,6 +39,29 @@ def print(message)
   puts message.colorize(:yellow)
 end
 
+def get_archive_url(version)
+  'https://github.com/xcodeswift/xctools/archive/' + "#{version}" + '.tar.gz'
+end
+
+def download_archive(url)
+  # filedownload, follow link, output errors, no progress meter
+  `curl -OLSs #{url}`
```
Comment (pepibumur) [inline Rakefile]: What about passing the local name *(e.g. xctools.tar.gz)* and git ignore it to prevent that we include it in the repository?

**Comment by yonaskolb** [comment]
Awesome! I’ll be copying this and adding it to my make files for my other open source projects.
What do you guys think about rake vs make? I don’t really have an opinion, I just defaulted to make

**Comment by pepibumur** [comment]
Great @yonaskolb 👍 . Personally I prefer Rakefile because I'm more familiar with the Ruby syntax. But If you don't have any complex scripting, Makefile should be ok as well!

**Comment by yonaskolb** [comment]
Just a comment, even though I saw you just made the version change. The file could can be removed after it’s used, that means you don’t need anything in gitignore and you also don’t end up with a file in your directory. Then using a fixed file name just becomes a question of how portable the script is

**Comment by alpreu** [comment]
@yonaskolb You're right, I guess it is also better to remove the file so in case the download fails it does not update the Formula to stale data from a previous download. I will change it again and add the rake task :)

**Comment by yonaskolb** [comment]
Yeah, that’s another reason I guess it’s good to keep the version name in the file, so that doesn’t happen by accident in any other way

**Pull request merged**